models/cow_protocol/ethereum/cow_protocol_ethereum_schema.yml

```
@@ -0,0 +1,25 @@
+version: 2
+
+models:
+  - name: cow_protocol_ethereum_events
```

Not sure, but I think this should be...
```suggestion
  - name: cow_protocol_ethereum_solvers
```

---

Yes, that's right ✔️